frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusCard.tsx

```diff
@@ -55,7 +55,12 @@ function cardContent({ robot }: RobotProps) {
                     <RobotStatusChip status={robot.status} />
                 </VerticalContent>
                 <VerticalContent $alignItems="end">
-                    <PressureStatusView pressure={robot.pressureLevel} />
+                    <PressureStatusView
+                        pressure={robot.pressureLevel}
+                        pressureStatus={PressureStatus.Normal}
+                        upperPressureWarningThreshold={robot.model.upperPressureWarningThreshold ?? 80}
+                        lowerPressureWarningThreshold={robot.model.lowerPressureWarningThreshold ?? 40}
```

Comment:
Should this be where the default value lives? should be defined somewhere and not in the function

Reply:
This value will only be utilized incase the value is somehow null. It is the same value in Initdb.cs